lib/index.js

```diff
@@ -36,7 +36,7 @@ exports.createSecret = function () {
  */
 exports.createCredentials = function (args) {
   var secret = args.secret || required('secret')
-    , ttl = args.ttl || DAY_IN_SECONDS
+    , ttl = args.ttl || DAY_IN_MILLISECONDS
```

Comment:
The `ttl` argument is in seconds, so the multiplication must occur on the result of this expression.